src/components/simulation/SimulationCodeEditor.vue

```diff
@@ -46,6 +120,62 @@ export default Vue.extend({
 
     const state = reactive({
       code: props.code as ProjectCode,
+      blockItems: [
+        {
+          disabled: false,
+          icon: 'mdi-delete-empty-outline',
+          icontext: 'reset',
+          text: 'Reset kernel',
+          value: 'resetKernel',
+        },
+        {
+          disabled: false,
+          icon: 'mdi-debug-step-into',
+          icontext: 'insite',
+          text: 'Run Simulation with Insite',
```

Comment:
I'd like to keep this consistent with the other titles.
```suggestion
          text: 'Run simulation with Insite',
```